zipline/finance/slippage.py

```diff
@@ -49,27 +49,29 @@ def volume_for_bar(self):
         return self._volume_for_bar
 
     @abc.abstractproperty
-    def process_order(self, trade_bar, order):
+    def process_order(self, bar_data, price, volume, order):
```

Comment:
Unrelated to this PR, but it looks like this whole file is indented one space?  That's not the case on master.